### PR TITLE
Support chinese traditional

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

Requires https://github.com/opentripplanner/otp-ui/pull/741.

This PR adds support for Chinese Traditional (`zh-Hant` in config, `zh_Hant.yml` for the content), including translated content from #1243 and code changes needed to enable that language. The Chinese (Simplified) file is renamed to `zh_Hans.yml`.

Config changes:
- `zh` is still supported by itself, i.e. if no other Chinese languages are configured such as Chinese (Traditional).
- If multiple Chinese languages/scripts are configured, then Chinese (Simplified), if used, must be configured as `zh-Hans`.
- Same applies for other languages.

## PR Checklist
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/actions/ui.js [hunk 1]
```diff
@@ -453,9 +453,7 @@ export function setLocale(locale) {
       dispatch(updateLocale({ locale: matchedLocale, messages }))
 
       // Update the lang attribute in the root <html> element.
-      // (The lang is the first portion of the locale.)
-      const lang = matchedLocale.split('-')[0]
-      document.documentElement.lang = lang
+      document.documentElement.lang = matchedLocale
```
Comment (binh-dam-ibigroup) [inline lib/actions/ui.js:456]: The HTML `lang` tag can be a locale (language-suffix format) so we should be good here.

**Review by amy-corson-ibigroup — APPROVED** [review]
Packages updated and looking good to me!!

**Review by josh-willis-arcadis — APPROVED** [review]
LGTM!

**Comment by binh-dam-ibigroup** [comment]
> Packages updated and looking good to me!!

Thanks for updating the OTP-UI packages!

**Pull request merged**